alerting/src/main/kotlin/org/opensearch/alerting/model/destination/DestinationContextFactory.kt

```diff
@@ -21,15 +21,16 @@ import org.opensearch.common.xcontent.NamedXContentRegistry
 class DestinationContextFactory(
     val client: Client,
     val xContentRegistry: NamedXContentRegistry,
-    private var destinationSettings: Map<String, SecureDestinationSettings>
+    private var destinationSettings: Map<String, SecureDestinationSettings>,
+    private val clusterName: String
```

Comment:
We might not need the `clusterName` portion. It doesn't look like it's being used here.